bin/mink-test-server

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# fail hard
+set -o pipefail
+# fail harder
+set -eu
+
+base_dir=$(dirname "$(dirname "$(readlink -f "$0")")")
+host="localhost:8002"
+php_bin=${MINK_PHP_BIN-php}
```

Comment:
the ``-`` is about defining a default value when the env var is not defined. Everything after it is this default value.
The goal of ``MINK_PHP_BIN`` is to be able to force a specific PHP runtime to be used. But we default to using the php available in the PATH.

I thought about using options rather than env vars, which may be easier to use for devs, but env vars are much easier to handle in Bash than introducing the parsing of options, so I went this way for now.
I also thought about making the host configurable, but decided it is not worth it for now (especially given that a hardcoded one allows to use it by default in the testsuite)